COMPATIBILITY.md

```diff
@@ -5,23 +5,15 @@
 
 The below matrix shows the compatibility of the [`opensearch-py`](https://pypi.org/project/opensearch-py/) with versions of [`OpenSearch`](https://opensearch.org/downloads.html#opensearch).
 
-| OpenSearch Version | Client Version |
-| --- | --- |
-| 1.0.0 | 1.0.0 |
-| 1.0.1 | 1.0.0 |
-| 1.1.0 | 1.1.0 |
-| 1.2.0 | 1.1.0 |
-| 1.2.1 | 1.0.0 |
-| 1.2.2 | 1.0.0 |
-| 1.2.3 | 1.0.0 |
-| 1.2.4 | 1.0.0 |
-| 1.3.0 | 1.1.0 |
-| 1.3.1 | 1.1.0 |
-| 1.3.2 | 1.1.0 |
-| 1.3.3 | 1.1.0 |
-| 2.0.0 | 2.0.0 |
-| 2.0.1 | 2.0.0 |
+Compatibility 
+
+| Client Version | OpenSearch Version | Notes |
+| --- | --- | --- |
+| 1.0.0 | 1.0.0-1.0.1, 1.2.1-1.2.4 | |
```

Comment:
It might be a typo in the previous markdown, 1.1.0 client version was released with being compatible from OpenSearch versions 1.3.0. Can you please update that?

Reply:
@VachaShah Made the changes as suggested.